server/src/controllers/campaign.js

```diff
@@ -61,9 +81,24 @@ class CampaignController {
               [
                 db.Sequelize.literal(`(
                   SELECT COUNT(*) FROM "StepProspects" 
-                  WHERE "StepProspects"."stepId" = "Steps"."id" AND "StepProspects"."currentStep" = true)`),
+                  WHERE "StepProspects"."stepId" = "Steps"."id" 
+                  AND "StepProspects"."currentStep" = true)`),
                 "prospectCount",
               ],
+              [
+                db.Sequelize.literal(`(
+                  SELECT COUNT(*) FROM "StepProspects" 
+                  WHERE "StepProspects"."stepId" = "Steps"."id" 
+                  AND "StepProspects"."replied" = true)`),
+                "repliedCount",
+              ],
+              [
+                db.Sequelize.literal(`(
+                  SELECT COUNT(*) FROM "StepProspects" 
+                  WHERE "StepProspects"."stepId" = "Steps"."id" 
+                  AND "StepProspects"."contacted" = true)`),
+                "contactedCount",
+              ],
```

Comment:
This is not the most efficient approach, you could probably get all these counts from one SQL Query using a Group By and Count statement 

Something like this:
https://stackoverflow.com/questions/32595456/sql-find-total-count-of-each-type-in-a-column

I think this is fine right now since it's super clean and easy to read, and efficiency is not an issue at the moment. This is something to just note for the future